src/chapters/interfaces-and-polymorphism/exercises.rst

```diff
@@ -0,0 +1,202 @@
+Exercises: Interfaces and Polymorphism
+=======================================
+
+It takes time for new Java coders to recognize the usefulness of interfaces. At
+first glance, they do not seem to provide much benefit over extending a base
+class, adding instance methods to a class, or overriding a method like
+``toString``.
```

Comment:
Move out of the third person and into the first person to make this intro seem more inviting.

Reply:
Done.